app/src/main/java/org/simple/clinic/removeoverdueappointment/RemoveOverdueEffectHandler.kt

```diff
@@ -54,6 +55,16 @@ class RemoveOverdueEffectHandler @AssistedInject constructor(
     }
   }
 
+  private fun markPatientAsRefusedToComeBack(): ObservableTransformer<MarkPatientAsRefusedToComeBack, RemoveOverdueEvent> {
+    return ObservableTransformer { effects ->
+      effects
+          .observeOn(schedulersProvider.io())
+          .doOnNext { patientRepository.updatePatientStatusToMigrated(it.patientId) }
```

Comment:
Did we confirm it with server/product folks that patients who are refusing to comeback are marked as migrated?

Reply:
Yes. Please see the conversation over [here](https://simpledotorg.slack.com/archives/C013HS4N9U3/p1677745160716859?thread_ts=1674733894.784699&cid=C013HS4N9U3)

Reply:
Yes. Please see the conversation over [here](https://simpledotorg.slack.com/archives/C013HS4N9U3/p1677745160716859?thread_ts=1674733894.784699&cid=C013HS4N9U3)